index.html

```diff
@@ -185,5 +185,25 @@ <h2 class="third-text"><a>let's connect</a></h2>
           </li>
         </ul>
     </section>
+    <section class="contact-form">
+        <ul>
+            <li><img class="ellipse-12-2" src="elements/ellipse-12-2.png" alt="icon-7"></li>
+            <li><img class="ellipse-5-w" src="elements/ellipse-5-white.png" alt="icon-8"></li>
+            <li><img class="ellipse-3-w" src="elements/ellipse-3-white.png" alt="icon-9"></li>
+            <li><img class="rect-4-d" src="elements/rectangle-4-dark.png" alt="icon-10"></li>
+            <li><img class="rect-7-2" src="elements/rectangle-7-2.png" alt="icon-11"></li>
+        </ul>
```

Comment:
- [ ] As requested by the previous code reviewer, please make sure to attach a background image to the contact section instead of hard coding it with HTML and CSS. This way is better because it's reducing all of this code to a single CSS line.
Hint:
`
.section {
    background-image: url("image_path_here");
} 
`